sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -513,10 +512,14 @@ kernel_pid_t gnrc_ipv6_netif_find_by_prefix(ipv6_addr_t **out, const ipv6_addr_t
         mutex_unlock(&(ipv6_ifs[i].mutex));
     }
 
+    if (out != NULL) {
+        *out = found;
```

Comment:
My motivation is that when:

- `ENABLE_DEBUG` is 1,
- `out` (not `*out`) is `NULL`, and
- we found a match,

then we should print a debug message at line 522 without crashing. The old implementation dereferences `out` unconditionally, resulting in a null pointer dereference.